Framework/DataHandling/src/LoadNexusLogs.cpp

```diff
@@ -106,10 +109,12 @@ void LoadNexusLogs::exec() {
   std::string filename = getPropertyValue("Filename");
   MatrixWorkspace_sptr workspace = getProperty("Workspace");
 
+  std::string entry_name = getPropertyValue("NXentryName");
```

Comment:
Now that the user can define the value of `NXentryName` the exception on line 123 may be misleading.